### PR TITLE
Initial Release: 1.0.0 version of the website

### PR DESCRIPTION
**Pull request by isume295**
## What does this PR do?

This PR is raised to implement the following features 👍:

### API
- [x] Use an API so you can base the development of the webapp around it. 
- [x] Use this [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to record the different user interactions (likes and comments).

### Interface
- [x] Build *home page* and *popup* interfaces using the provided wireframes.

#### Home page
- [x] When the page loads, the web-app retrieves data from Involvement and Items APIs.
- [x] When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
- [x] When the user clicks on the "Comments" button, the Comments popup appears.
- [x] Home page header and navigation similar to the given mockup.
- [x] Home page footer similar to the given mockup.

#### Comments popup
- [x] When the comment popup loads, the web-app retrieves data from the Involvement and Items APIs.
- [x] When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.

#### Counters
- [x] Count the number of items (home)
- [x] Count the number of comments (popup)

## Technical set up 🙌
- [x] Set up the repository on GitHub and use Gitflow.
- [x] Set up webpack.
- [x] Set up a JavaScript testing library (Jest).
- [x] Added the HTML for the structure. 
- [x] Added the SCSS file for the styling.
- [x] Added the js file for DOM manipulation.
- [x] Used the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import './style.scss';
+import init from './modules/init.js';
+
+await init();
```
Comment (elisha2kyakpo1) [inline src/index.js:4]: - [ ] The deployed app runs very well, but when I tried to run the app on my machine, I got the warnings on the screen. Kindly fix these errors for a good user experience. I have attached an image for your reference.

meal

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @isume295 and @Salimer, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights



## Required Changes ♻️
_
👏You have a very nice UI making it easy for me to do my work.

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Tests passing👏
👏 The app works as expected on the deployed link.

Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Salimer** [comment]
Thank you @elisha2kyakpo1 for the thorough review. We managed to fix all the warnings. 🥳

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 🎉  🎉  🎉  🎉  🎉 🎉 🎉 🎉 🎉 
Hi @isume295 & @Salimer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x]  Good UI design :white_check_mark:
- [x]  Application works as expected :white_check_mark: 
- [x]  Pull request has a descriptive title and summary :white_check_mark: 
- [x]  Repository has a professional readme file :white_check_mark: 
- [x]  Git flow is used correctly :white_check_mark: 
- [x]  Implemented the changes from the previous reviewer :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**